src/fragments/lib/auth/js/manageusers.mdx

```diff
@@ -166,24 +176,61 @@ let result = await Auth.updateUserAttributes(user, {
 });
 console.log(result); // SUCCESS
 ```
+<Callout>
+Note: If you change an attribute that requires confirmation (i.e. email or phone number), you will receive a confirmation code to that attribute.
+</Callout> 
 
-You can delete user attributes:
+`Auth.updateUserAttributes()` function dispatches hub event to help identify attributes that require
+verification. You can listen to `updateUserAttributes` event on `auth` channel:
 
 ```javascript
-let user = await Auth.currentAuthenticatedUser();
+Hub.listen('auth', ({payload}) => {
+    if (payload.event === 'updateUserAttributes') {
+        const resultObject = payload.data;
+    }
+});
+```
+You can learn more about how to listen to Hub events [here](https://docs.amplify.aws/lib/utilities/hub/q/platform/js/).
 
-let result = await Auth.deleteUserAttributes(user, ['family_name']);
-console.log(result); // SUCCESS
+Example of the dispatched result object:
+
+```javascript
+{
+    'email': {
+        isUpdated: false // indicates that attribute requires verification
+        codeDeliveryDetails: {
+            AttributeName: 'email',
+            DeliveryMedium: 'EMAIL',
+            Destination: 'm***@a...'
+        }
+    },
+    'family_name': {
+        isUpdated: true
+    }
+}
 ```
+If user sends several attributes for update and one or more of them is `read_only` or doesn't exist, Amplify JS library will throw
```

Comment:
If a user sends several attributes for update and one or more of them is `read_only` or doesn't exist, the Amplify JavaScript library will throw